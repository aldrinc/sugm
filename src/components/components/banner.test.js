import React from 'react';
import Banner from './banner';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';


it('banner renders correctly', () => {
    const tree = renderer.create(
      <Router>
        <Banner/>
      </Router>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });

