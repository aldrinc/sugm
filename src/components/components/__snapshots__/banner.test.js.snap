// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`banner renders correctly 1`] = `
<div
  className="container-fluid m_nopadding"
>
  <div
    className="row"
  >
    <div
      className="app_banner_cnt"
    >
      <img
        alt="Banner_img"
        src="banner_img.jpg"
      />
      <img
        alt="Banner_img"
        className="mob_img"
        src="mobile_banner.jpg"
      />
      <div
        className="app_banner_text"
      >
        <h1>
          The thrill of living is in the joy of giving.
        </h1>
        <a
          href="/best-selling"
          onClick={[Function]}
        >
          Shop our best sellers and more here
        </a>
      </div>
    </div>
  </div>
</div>
`;
